content/resources/faq/_index.md

```diff
@@ -0,0 +1,184 @@
+# Frequently Asked Questions
+
+## 1. Compose Services ![Compose Services](img/compose-services.svg)
+
+**I installed Compose Services, how do I create a program and a project?**
+
+Please refer to corresponding
+[Gen3 Documentation](https://gen3.org/resources/operator/#5-programs-and-projects)
+to create your first program and project.
+
+
+**I created a program, but I see `NotFound` error when I try to create a
+project within my program.**
+
+Please make sure you inserted program `name` in the URL, not
+`dbgap_accession_number`.
+
+
+**I can not submit data to the project I just created.**
+
+Please ensure you granted yourself permissions in the `user.yaml` file. You can
+find this file in the `Secrets` folder within your Compose Services folder.
+Follow provided in the file example to grant yourself relevant permissions. You
+can find more information in the
+[Setting Up Users](https://github.com/uc-cdis/compose-services#setting-up-users)
+documentation.
+
+
+**User.yaml contains information about program/project, but I still can not
+access it.**
+
+Please ensure you are using `dbgap_accession_number` value as an identifier for
+the `auth_id` field, not program's `name` or project's `code`.
+
+
+**I see a warning about my connection is not being private.**
+
+Don't worry. The warning about connection not being private appears because
+`creds_setup.sh` script generated self-signed SSL certificate to allow
+microservices to communicate with each other without bypassing SSL verification.
+Your browser can not verify this certificate. Once you install a trusted SSL
+certificate instead of generated by our script, this warning will go away.
+
+
+**I want to run Compose Services on the server, how to configure
+ it.**
+
+To configure Compose Services with your domain, please change the `BASE_URL`
+field in the `fence_config.yaml` file, and the `hostname` field in the
+`peregrine.json`, `sheepdog.json` files.
```

Comment:
To configure Compose Services with your domain, please change the `BASE_URL`
field in the `fence_config.yaml` file, and the `hostname` field in the
`peregrine.json`and `sheepdog.json` files.

Reply:
Thanks! Updated in 7990669

Reply:
replace , with and